acquire/tools/decrypter.py

```diff
@@ -265,51 +265,64 @@ def check_existing(in_path: Path, out_path: Path, status_queue: multiprocessing.
 
 
 def worker(task_id, stop_event, status_queue, in_path, out_path, key_file=None, key_server=None, clobber=False):
+    success = False
+    message = "An unknown error occurred"
+
     try:
         if check_existing(in_path, out_path, status_queue) and not clobber:
+            message = "Output file or decompressed file already exists!"
             return
 
         _update(status_queue, task_id, visible=True)
-        success = False
         with in_path.open("rb") as infh:
             try:
                 ef = EncryptedFile(infh, key_file=key_file, key_server=key_server)
             except Exception as e:
-                _info(status_queue, f"{in_path} • Error opening encrypted file: {e}")
+                message = f"Error opening encrypted file: {e}"
+                _info(status_queue, f"{in_path} • {message}")
+                return
             _info(
                 status_queue,
                 f"{in_path} • File: {ef.file_header.magic.decode()} | {ef.file_header.header_type} | {ef.timestamp}",
             )
             _info(status_queue, f"{in_path} • Header: {ef.header.magic.decode()} | {ef.header.cipher_type}")
             _update(status_queue, task_id, total=ef.size)
 
-            with out_path.open("wb") as outfh:
-                _info(status_queue, f"{in_path} • Decrypting to {out_path}")
-                _start(status_queue, task_id)
-                try:
+            try:
+                with out_path.open("wb") as outfh:
+                    _info(status_queue, f"{in_path} • Decrypting to {out_path}")
+                    _start(status_queue, task_id)
+
                     for chunk in ef.chunks():
                         if stop_event.is_set():
                             raise ValueError("stopping")
                         outfh.write(chunk)
                         _update(status_queue, task_id, advance=len(chunk))
                     ef.verify()
-                    _info(status_queue, f"{in_path} • File verified OK!")
+                    message = "File verified OK!"
+                    _info(status_queue, f"{in_path} • {message}")
                     success = True
-                except ValueError as e:
-                    _info(status_queue, f"{in_path} • Error decrypting file: {e}")
-                except VerifyError as e:
-                    _info(status_queue, f"{in_path} • Verify error: {e}")
-                    _info(status_queue, f"{in_path} • ! Output file should NOT be trusted")
-                except Exception as e:
-                    _info(status_queue, f"{in_path} • Unknown error: {e}")
+            except ValueError as e:
+                message = f"Error decrypting file: {e}"
+                _info(status_queue, f"{in_path} • {message}")
+            except VerifyError as e:
+                _info(status_queue, f"{in_path} • Verify error: {e}")
+                message = "! Output file should NOT be trusted"
```

Comment:
Maybe add something like `Verification failed` to the message, so it is clear why the output should not be trusted.